### PR TITLE
Update messaging for capistrano deployments

### PR DESCRIPTION
**Pull request by zoiec**
🐘

- added titleize to messaging
- titleize is a part of rails, required rails
- changed format of message so that environment is last

### CODE REVIEW

File: capistrano-deploy.gemspec [hunk 1]
```diff
@@ -12,6 +12,7 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
 
+  s.add_dependency('rails', '>= 3.0.0')
```
Comment (jessedoyle) [inline capistrano-deploy.gemspec]: It looks like we're adding `rails` as a dependency to get the `titleize` method?

What do you think about adding ActiveSupport instead?

i.e.

```ruby
# in gemspec
s.add_dependency 'activesupport'

# in lib/capistrano-deploy.rb
require 'active_support/all'
```

**Review by jessedoyle — CHANGES_REQUESTED** [review]
Just a question in the review.

**Comment by jessedoyle** [comment]
:hankey:

**Comment by zoiec** [comment]
👋

**Review by jessedoyle — APPROVED** [review]

**Comment by jessedoyle** [comment]
:elephant::elephant::elephant:

**Pull request merged**